### ISSUE TITLE
Consolidate images

### ISSUE THREAD

**Issue by siznax** [siznax/wptools#33]
Each image should be put in a dict with its key indicating the source, and no attempt to compute a media URL until later. Something like this:

```
images {
  get_query: <filename>,
  get_parse: <filename>,
  get_wikidata: [<filename],
  get_rest: <filename>
}
```

We should open another issue to provide a method to resolve filenames.

**Comment by siznax** [comment]
Implemented with 29a506332ad456aa06d586fcf5bac6e51604d42f:

```python
>>> a = wptools.page("Alice's Adventures in Wonderland").get()
>>> a.images
{'parse-Cover': None,
 'parse-image': 'File:Alice_in_Wonderland,_cover_1865.jpg',
 'query-pageimage': u"Alice's_Adventures_Under_Ground_-_Lewis_Carroll_-_British_Library_Add_MS_46700_f45v.jpg",
 'query-thumbnail': {u'height': 240,  u'source': u'https://upload.wikimedia.org/wikipedia/commons/thumb/6/61/Alice%27s_Adventures_Under_Ground_-_Lewis_Carroll_-_British_Library_Add_MS_46700_f45v.jpg/153px-Alice%27s_Adventures_Under_Ground_-_Lewis_Carroll_-_British_Library_Add_MS_46700_f45v.jpg', u'width': 153},
 'rest-image': {u'file': u"Alice's Adventures Under Ground - Lewis Carroll - British Library Add MS 46700 f45v.jpg", u'height': 750, u'width': 479},
 'rest-thumb': {u'height': 1024, u'url': u'//upload.wikimedia.org/wikipedia/commons/6/61/Alice%27s_Adventures_Under_Ground_-_Lewis_Carroll_-_British_Library_Add_MS_46700_f45v.jpg', u'width': 654},
 'wikidata-image': u'AlicesAdventuresInWonderlandTitlePage.jpg'}
```

**Comment by siznax** [comment]
Resolving `file` attributes to `url`s will be tracked in #34
